README.md

```diff
@@ -115,103 +105,46 @@ con = glaredb.connect("glaredb://user:pass@host:port/deployment")
 con.sql("select 'hello hybrid exec';").show()
 ```
 
-GlareDB work with [Pandas] and [Polars] DataFrames out of the box:
-
-```python
-import glaredb
-import polars as pl
-
-df = pl.DataFrame(
-    {
-        "A": [1, 2, 3, 4, 5],
-        "fruits": ["banana", "banana", "apple", "apple", "banana"],
-        "B": [5, 4, 3, 2, 1],
-        "cars": ["beetle", "audi", "beetle", "beetle", "beetle"],
-    }
-)
-
-con = glaredb.connect()
+GlareDB also work with [Pandas] and [Polars] DataFrames out of the box:
 
-df = con.sql("select * from df where fruits = 'banana'").to_polars();
-
-print(df)
+```py
+glaredb.sql("select * from pd_df").to_pandas()
+glaredb.sql("select * from pl_df").to_polars()
 ```
 
 ### Local server
 
-The `server` subcommand can be used to launch a server process for GlareDB:
-
 ```shell
+# The `server` subcommand can be used to launch a server process for GlareDB:
 ./glaredb server
 ```
 
-To see all options for running in server mode, use `--help`:
-
-```sh
-./glaredb server --help
-```
+> [!NOTE]
+> When launched as a server process, GlareDB can be reached on port `6543` using a
+> Postgres client. The following example uses `psql` to connect to a locally
+> running serve:
 
-When launched as a server process, GlareDB can be reached on port `6543` using a
-Postgres client. The following example uses `psql` to connect to a locally
-running server:
 
 ```shell
+# Connect to the server using psql client
 psql "host=localhost user=glaredb dbname=glaredb port=6543"
 ```
 
-## Your first data source
```

Comment:
I wouldn't drop this (and might even move it above 'running in server mode') as it's our main chance to demonstrate what glaredb is and what makes it cool.